@here/olp-sdk-dataservice-read/test/unit/StreamLayerClient.test.ts

```diff
@@ -155,4 +157,164 @@ describe("StreamLayerClient", () => {
                 assert.equal(mockedErrorResponse, error.statusText);
             });
     });
+
+    it("Should method getData provide data", async () => {
+        const mockedBlobData: Response = new Response("mocked-blob-response");
+        const mockedMessage = {
+            metaData: {
+                partition: "314010583",
+                checksum: "ff7494d6f17da702862e550c907c0a91",
+                compressedDataSize: 152417,
+                dataSize: 250110,
+                data: "",
+                dataHandle:
+                    "iVBORw0KGgoAAAANSUhEUgAAADAAAAAwBAMAAAClLOS0AAAABGdBTUEAALGPC/xhBQAAABhQTFRFvb29AACEAP8AhIKEPb5x2m9E5413aFQirhRuvAMqCw+6kE2BVsa8miQaYSKyshxFvhqdzKx8UsPYk9gDEcY1ghZXcPbENtax8g5T+3zHYufF1Lf9HdIZBfNEiKAAAAAElFTkSuQmCC",
+                timestamp: 1517916706
+            },
+            offset: {
+                partition: 7,
+                offset: 38562
+            }
+        };
+        getBlobStub.callsFake(
+            (builder: any, params: any): Promise<Response> => {
+                return Promise.resolve(mockedBlobData);
+            }
+        );
+
+        const response = await streamLayerClient.getData(mockedMessage);
+        assert.isDefined(response);
+        console.log(response);
```

Comment:
Please remove `const response =`,  and 
`        assert.isDefined(response);
        console.log(response);`
we don't need that there